e2e/tests/hmctsAdministersCaseAfterSubmission_@slow_test.js

```diff
@@ -142,7 +142,7 @@ Scenario('HMCTS admin edits supporting evidence document on C2 application', asy
   caseViewPage.selectTab(caseViewPage.tabs.otherApplications);
   I.seeInTab(['Additional applications 1', 'C2 application', 'Supporting documents 1', 'Document name'], 'Updated document name');
   I.seeInTab(['Additional applications 1', 'C2 application', 'Supporting documents 1', 'Notes'], 'This is a note about supporting doc');
-  I.seeInTab(['Additional applications 1', 'C2 application', 'Supporting documents 1', 'Date and time uploaded']);
+  I.seeTextInTab(['Additional applications 1', 'C2 application', 'Supporting documents 1', 'Date and time uploaded']);
```

Comment:
Also odd that these need changing

Reply:
hmm, not sure since we moved it into the helper seems to not work the old way

Reply:
Not sure how it was working the old way tbh, this change is making it the right way